content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -526,35 +526,35 @@ To set a different service name per Ingress using annotations:
 The above overrides the default `nginx-ingress-controller.ingress-nginx` service name.
 
 ### Ingress Controller Sampling
-The Ingress-NGINX Controller for Kubernetes uses `dd-opentracing-cpp` as its
-underlying Datadog tracing library.
+To set a fixed sampling rate, use the [datadog-sample-rate][16] option in the
+ingress controller's [ConfigMap][17]. For example, to set the sampling rate to
+40%:
 
-To control the volume of Ingress Controller traces that are sent to Datadog,
-specify a sampling rule that matches all traces. The `sample_rate` configured
-in the rule determines the proportion of traces that are sampled. If no
-rules are specified, then sampling defaults to 100%.
-
-Specify sampling rules by using the `DD_TRACE_SAMPLING_RULES` environment
-variable. To define sampling rules in the Ingress Controller:
-
-1. Instruct NGINX to forward the environment variable to its worker processes by adding the following [main-snippet][14] to the `data` section of the Ingress Controller's `ConfigMap`:
-   ```yaml
-   data:
-     main-snippet: "env DD_TRACE_SAMPLING_RULES;"
-   ```
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/instance: ingress-nginx
+    app.kubernetes.io/name: ingress-nginx
+    app.kubernetes.io/part-of: ingress-nginx
+    app.kubernetes.io/version: 1.7.1
+  name: ingress-nginx-controller
+  namespace: ingress-nginx
+data:
+  datadog-collector-host: $HOST_IP
+  enable-opentracing: "true"
+  datadog-sample-rate: 0.4
+```
 
-2. Specify a value for the environment variable in the `env` section of the Ingress Controller's `Deployment`. For example, to keep 10% of traces originating from the Ingress Controller:
-   ```yaml
-   env:
-   - name: DD_TRACE_SAMPLING_RULES
-     value: '[{"sample_rate": 0.1}]'
-   ```
-   To use the [Datadog Agent calculated sampling rates][10] (10 traces per second per Agent by default), specify an empty array of sampling rules:
-   ```yaml
-   env:
-   - name: DD_TRACE_SAMPLING_RULES
-     value: '[]'
-   ```
+<div class="alert alert-warning">
+Due to a bug in the Datadog tracing integration, the <a
```

Comment:
How widespread if this? We don't typically list bugs in our docs. Is this something that we can add to the support docs instead?

Reply:
Thanks @dgoffredo, here's my suggested change. If this is good to go I can merge!

```suggestion
The <a href="https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#datadog-priority-sampling">datadog-priority-sampling</a> option has no effect, and it is not possible to use the sampling rates <a href="https://docs.datadoghq.com/tracing/trace_pipeline/ingestion_mechanisms/#in-the-agent">calculated by the Datadog Agent</a>. 
```

Reply:
This is because of a bug that we introduced and are working to fix. The way you suggest, it sounds like we are making a claim contrary to the [upstream project's documentation][1].

Our documentation describes the product and its use. This part of the product has a bug, so let's mention it here.

[1]: https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#datadog-priority-sampling

Reply:
Thanks for the context @dgoffredo, will merge as is!